src/desktop/mainwindow.cpp

```diff
@@ -869,6 +884,34 @@ void MainWindow::writeSettings()
 	m_dockToolSettings->saveSettings();
 }
 
+void MainWindow::requestUserInfo(int userId)
+{
+	net::Client *client = m_doc->client();
+	QJsonObject info{{"type", "request_user_info"}};
+	client->sendMessage(drawdance::Message::makeUserInfo(
+		client->myId(), userId, QJsonDocument{info}));
+}
+
+void MainWindow::sendUserInfo(int userId)
```

Comment:
I see. Hopefully the UI changes to the input settings will help with this a bit since there will be a tablet tester right there. I also feel like I would prefer a “Send diagnostics” and/or a thing in the Help menu for a user to self-resolve the problem since I would imagine a technical user may not always be available to help, but I might also just be acting too old fashioned.

Reply:
Usually the problem is that there's too many people trying to help simultaneously! Improving the way the tablet settings work should help, plus not having to restart the application after tablet input changes will reduce the friction by a lot. For now I'm not willing to let go of it though, experience has just been too negative in that regard.